spec/factories/users.rb

```diff
@@ -44,6 +45,13 @@
         user.save!
       end
     end
+
+    factory :disabled_user do
+      after(:create) do |user|
+        user.is_disabled = true
+        user.save!
+      end
+    end
```

Comment:
instead of a factory, you can use a trait here (not tested):

```ruby
trait :disabled do
  is_disabled true
end
```

And then in your test: `let(:disabled_user) { create(:user, :disabled) }`

I'm not sure why the `after :create` block is needed? Can't the `is_disabled` attribute be set like in a normal factory, as I show above?